.github/workflows/release_test.yml

```diff
@@ -0,0 +1,34 @@
+name: release test
+on:
+  push:
+    branches:
+      - master
+      - release-*
+    tags:
+      - 'v*.*.*'
+  pull_request:
+    paths:
+      - .github/workflows/release_test.yml
+      - '.goreleaser*.yml'
+      - go.mod
+      - go.sum
```

Comment:
Why do we need to run goreleaser test with go.mod/go.sum updates?

Reply:
https://github.com/reviewdog/reviewdog/blob/f15827cc7b7ce1c79630a4f26e8ba9482c3979ad/.github/workflows/release_test.yml#L28

This is because `go.mod` is referenced above, and I think it would be better to check this CI after updating `go.mod`.

Reply:
Sounds good!
Thanks for the explanation :+1: